frontend/webpack.config.js

```diff
@@ -94,6 +95,6 @@ module.exports = {
     performance: {
         hints: false
     },
-    devtool: '#eval-source-map',
+    devtool: '#cheap-source-map',
```

Comment:
Cut `app.js` bundle size from ~10m to ~1.8m

Reply:
Very nice. :sunglasses: